### PR TITLE
py-distributed: Update to the latest version (2020.12.0)

### PR DESCRIPTION
**Pull request by a-saitoh-fj**
py-distributed: Update to the latest version (2020.12.0)

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-distributed/package.py [hunk 1]
```diff
@@ -10,8 +10,9 @@ class PyDistributed(PythonPackage):
     """Distributed scheduler for Dask"""
 
     homepage = "https://distributed.dask.org/"
-    pypi = "distributed/distributed-2.10.0.tar.gz"
+    url      = "https://pypi.io/packages/source/d/distributed/distributed-2.10.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-distributed/package.py]: Undo this change

**Comment by a-saitoh-fj** [comment]
As pointed out, I changed the description of url back to the description of pypi.

**Review by alalazo — APPROVED** [review]
